### ISSUE TITLE
PID constants

### ISSUE THREAD

**Issue by JadBatmobile** [TheChosenZygote/irobotcreate_pid#1]
Hello,

In pid_control_node2, i see you are using the ki_lin variable in the angular PID computation.

Should this not be ki_ang?

Am i missing something?

Jad

**Comment by TheChosenZygote** [comment]
You are correct

**Comment by JadBatmobile** [comment]
thanks. I also see that the control takes x-y waypoint as destination, but not yaw as a final desired orientation. That is true right?

Its not too hard to add a little code to spin in place when the destination is reached to match a desired final yaw.

**Comment by TheChosenZygote** [comment]
Yes that is also correct. This was a short summer project and I won't be
updating this, so feel free to tinker it to your desire.
